sql/item_strfunc.cc

```diff
@@ -5414,7 +5414,8 @@ longlong Item_func_wsrep_sync_wait_upto::val_int()
       }
       else if (wait_gtid_ret == ENOMEM)
       {
-        my_error(ER_OUTOFMEMORY, MYF(0), func_name());
+        /* length is bogus here, no idea how much it tried */
+        my_error(ER_OUTOFMEMORY, MYF(0), WSREP_GTID_STR_LEN);
         ret= 0;
       }
     }
```

Comment:
You could also use here instead the error ER_OUT_OF_RESOURCES (which doesn't take the size).